core/server/index.js

```diff
@@ -44,7 +44,7 @@ function initialiseServices() {
         scheduling.init({
             schedulerUrl: config.get('scheduling').schedulerUrl,
             active: config.get('scheduling').active,
-            apiUrl: urlService.utils.urlFor('api', {version: 'deprecated'}, true),
+            apiUrl: urlService.utils.urlFor('api', {version: 'deprecated', versionType: 'content'}, true),
```

Comment:
🤔 maybe we miscommunicated, but shouldn't this be none explicit too?

Reply:
Hmm, my take was to update only cases which are already explicit for v0.1 and will not change. Since scheduling is not yet refactored into v0.1 specific right now and will need a refactor for v2 , like tests, so didn't update it 🤔If we are instead updating based on wherever we have `version: deprecated`, then it would make sense to update here as well as tests to non explicit. Should we do the latter and update everywhere where its `version:deprecated`?

Reply:
Discussed. We keep it for now 👍